source/_docs/composer.md

```diff
@@ -1,175 +1,87 @@
 ---
-title: Managing Custom Code with Composer
-description: Understand how to manage custom plugins and themes for WordPress sites using Composer.
-tags: [workflow]
-categories: []
+title: Composer Fundamentals and Workflows
+description: Start with Composer basics then explore suggested workflows for WordPress and Drupal sites on Pantheon.
+tags: [automation, workflow]
 ---
-Extending WordPress or Drupal with custom code is a common part of the development lifecycle. In order to optimize workflows, it's essential that custom plugins, modules, and themes are maintained independently from your sites and projects so they're easily reused.
+Composer is a PHP dependency manager that provides an alternative, more modern way to manage the external code used by a WordPress or Drupal site. At it's primary level, Composer needs:
+
+- A list of dependencies
+- A place to put the dependencies
+
+Understanding how Composer can be used independent of Drupal or WordPress is a good place to learn more about the general concepts. For a summary of basic usage, see [Composer's own documentation](https://getcomposer.org/doc/01-basic-usage.md).
+
+## Dependencies and Packages
+Composer encourages a mental model where code not written specifically for a given project is considered a "dependency". Dependencies for WordPress and Drupal include core, plugins, contrib modules, themes, and libraries. A single dependency, such as a theme, is referred to as a "package".
+
+By default, Composer can only see packages listed on [The PHP Package Repository](https://packagist.org/) which do not include Drupal or WordPress packages. Additional repositories must be configured for Composer to use packages not found in the default repository. Each framework provides it's own respective package repository so dependencies can be managed with Composer:
+
+- WordPress: [https://wpackagist.org](https://wpackagist.org)
+- Drupal 8: [https://packages.drupal.org/8](https://packages.drupal.org/8)
+- Drupal 7: [https://packages.drupal.org/7](https://packages.drupal.org/7)
+
+Site's created from Pantheon's example repositories already include the appropriate package repository within the `composer.json` file.
+
```

Comment:
We might want to include an example right here that illustrates how to add a package repository to a composer.json.